proto/batch_plan.proto

```diff
@@ -148,6 +148,14 @@ message TableFunctionNode {
   data.DataType return_type = 3;
 }
 
+message LookupJoinNode {
+  plan_common.JoinType join_type = 1;
+  repeated int32 left_key = 2;
+  repeated int32 right_key = 3;
```

Comment:
The right side of `LookupJoinNode` should also contains table id and columns to construct `RowSeqScanNode`.